ide/gsf.codecoverage/nbproject/project.xml

```diff
@@ -187,6 +187,7 @@
                 </dependency>
             </module-dependencies>
             <friend-packages>
+                <friend>org.netbeans.modules.gradle.java</friend>
```

Comment:
Unrelated to this PR, but maybe this is a nice opportunity to turn this & co. friend APIs needed by this PR to "under development" ones.

Reply:
That would be nice.